src/pulse3D/plate_recording.py

```diff
@@ -248,31 +243,34 @@ def _load_magnetic_data(self):
         self.noise_filtered_magnetic_data: NDArray[(2, Any), int]
         if self.noise_filter_uuid is None:
             self.noise_filtered_magnetic_data = self.fully_calibrated_magnetic_data
+            self.force = self.noise_filtered_magnetic_data
```

Comment:
This will break compatibility with beta 1 H5 files that don't have a noise filter specified. I also don't think that `_load_magnetic_data` needs to be called at all for optical files, we can probably just the `force` attribute with the raw values provided

Reply:
We still need to remove this line

Reply:
I removed `_load_magnetic_data` from optical files, what line are you mentioning?

Reply:
I removed `_load_magnetic_data` from optical files, what line are you mentioning?

Reply:
The line that this comment is attached to `self.force = self.noise_filtered_magnetic_data`